src/insns/atomic_exceptions.adoc

```diff
@@ -6,6 +6,8 @@ Requires <<r_perm>> and <<w_perm>> in the authorising capability.
 +
 If <<c_perm>> is not granted then store the memory tag as zero, and load `cd.tag` as zero.
 +
+If <<em_perm>> and <<w_perm>> are both unset and `cd.tag` is set and the `cd` is not sealed, then an implicit <<ACPERM>> clearing <<w_perm>> and <<em_perm>> is performed to obtain the final permissions on `cd` (see <<LC>>).
```

Comment:
It is worth clarifying what happens if the capability is unsealed. I saw a note about it in an instruction listing, but we should write this more clearly in places like this. My guess was that we just clear the tag of the loaded cap, which wrong.